psydac_accelerate.py

```diff
@@ -0,0 +1,52 @@
+
+import argparse
+import os
+from subprocess import run as sub_run
+import shutil
+
+# Add Argument --language at the command psydac-accelerate
+parser = argparse.ArgumentParser(
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    description="Get language for the pyccelisation."
```

Comment:
This is the description of the command `psydac-accelerate`...

Reply:
I have changed the comment + I add a small text to explain the goal of this file.
https://github.com/pyccel/psydac/pull/305/commits/dae1195ba40b3dc9757c340658811457ea7d5064

Reply:
What I meant is that the string passed to the argument `description` is what gets printed when you ask for help on the command line. It should describe the script itself, something like `"Accelerate all computational kernels in Psydac using Pyccel (editable install only)"`.